README.md

```diff
@@ -1,14 +1,49 @@
-# soen341
-Project for SOEN 341
+# soen341: Software Process
+Winter 2022 group 9  -Dr. Tse-Hsun (Peter) Chen
+## Introduction:
+### Objective: 
+develop an e-commerce website using agile methodologies.
+### Description:
+An e-commerce website allows users to buy and sell goods and services over the internet rather than in person.
+using this website, a merchant can process orders, accept payments and manage shipping to customers.
+### Core features:
+there are two types of users for this website:
+   - Customer
+   - Seller
+   
+these users will be able to choose which kind they are when signing up for the website.
+A seller is able to:
+- Add items for sale 
+- Upload pictures for the items
 
+A user is able to:
+- Browse the items offered by sellers
+- Add a specific number of items to the cart
+- Remove items from the cart
+- See the prices and other fees
+- Place an order
+- Pay for the order
+- Cancel an order
 
 
+ ## Team Members:
+ 
+- Chad Buck - CHADBUCK13
+- Dany Khaled - danikh25
+- Aida Berhe- Negousse
+- Matin Tavakkoli- matavak
+- Henry zhixing Li - JoliopoLi 
+- Mathieu Trudeau - mathieutrudeau
+- Hawa Diallo - hawadi
+
+## Technologies:
+### Front end:
+- Django
+- CSS
+- HTML
+### Back end:
```

Comment:
Firebase for users, they can use authorization services such as Google, Facebook and Apple